src/pipecat/transports/network/small_webrtc.py

```diff
@@ -547,11 +559,11 @@ async def start(self, frame: StartFrame):
 
         await self._client.setup(self._params, frame)
         await self._client.connect()
+        await self.set_transport_ready(frame)
```

Comment:
In the other transports, we’ve typically placed it last, once everything inside the transport is set up. Is there a reason why we need it here in this case ?

Reply:
i honestly can't remember now why i changed this order but i'm pretty sure it mattered 🤔